service/mantle/shopify/ApiServices.xml

```diff
@@ -519,4 +519,74 @@
         </actions>
     </service>
 
-</services>
\ No newline at end of file
+    <service verb="queue" noun="SetInventorySystemMessage">
+        <description>Service to update the quantity available in stock for a product</description>
+        <in-parameters>
+            <parameter name="productId" required="true"/>
+        </in-parameters>
+        <actions>
+
+            <entity-find-one entity-name="mantle.product.ProductIdentification" value-field="shopifyInvenotoryItem">
+                <field-map field-name="productId" from="productId"/>
+                <field-map field-name="productIdTypeEnumId" value="PidtInventoryItemShopify"/>
+            </entity-find-one>
+
+            <if condition="!shopifyInvenotoryItem">
+                <return error="true"
+                        message="The productId ${productId} does not exist"/>
+            </if>
+
+            <set field="inventoryItemId" from="shopifyInvenotoryItem.idValue"/>
+
+            <!-- get the store id -->
+            <entity-find-one entity-name="moqui.service.message.SystemMessageRemote" value-field="systemRemoteIdentification">
+                <field-map field-name="systemMessageRemoteId" value="DEMO_SHOPIFY_SET_INVENTORY_LEVEL_REMOTE"/>

```

Comment:
Change DEMO_SHOPIFY_SET_INVENTORY_LEVEL_REMOTE for DEMO_SHOPIFY_SET_INVENTORY_REMOTE